src/renderers/WebGLRenderer.d.ts

```diff
@@ -396,6 +397,16 @@ export class WebGLRenderer implements Renderer {
 		activeCubeFaceIndex?: number
 	): void;
 
+	/**
+	 * Copies a region of the currently bound framebuffer into the selected mipmap level of the selected texture.
+	 * This region is defined by the size of the texture's mip level and starts at the input position.
+	 * 
```

Comment:
It seems there is a linting error in this line: `Trailing spaces not allowed  no-trailing-spaces`

Reply:
Yeah, I saw that too, but I can't find it. When I try http://zz85.github.io/mrdoobapproves/, After clicking auto-format, it just says "No :(".